src/test/resources/fixtures/hmh/hmh-insighted/caliperGradeEventScore.described.json

```diff
@@ -8,15 +8,15 @@
       "type": "GradeEvent",
       "id": "urn:uuid:{uuid is auto-generated}",
       "actor": {
-        "id": "urn:uuid:{uuid of the student who completed the assignment}",
-        "type": "Person"
+        "id": "urn:uuid:{uuid of the teacher who graded the assignment} | {software application that auto-graded the assignment}",
+        "type": "Person | SoftwareApplication"
       },
       "action": "Graded",
       "object": {
         "id": "urn:uuid:{uuid is auto-generated}",
         "type": "Attempt",
         "assignable": {
-          "id": "urn:uuid:{uuid of this assignment}",
+          "id": "urn:uuid:{uuid of this assignment (teacher)}",
```

Comment:
having teacher in brackets here is a bit misleading, assignable represents the digital resource the grade applies against not a person.

Reply:
In our system, assignment API makes this distinction between teacher and student assignment.
LAUSD consumer requested this for clarification.